package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "resin-typescript-skeleton",
+  "version": "0.0.1",
+  "description": "Skeleton template for a Resin TypeScript project",
+  "homepage": "https://github.com/resin-io/resin-typescript-skeleton#readme",
+  "preferGlobal": true,
+  "types": "build/**/*.d.ts",
+  "keywords": [
+    "resin",
+    "typescript"
+  ],
+  "author": "Heds Simons <hedley@resin.io>",
+  "license": "Apache-2.0",
+  "main": "./build/implement-me.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/resin-io/resin-typescript-skeleton.git"
+  },
+  "bugs": {
+    "url": "https://github.com/resin-io/resin-typescript-skeleton/issues"
+  },
+  "scripts": {
+    "build": "gulp build"
+  },
+  "devDependencies": {
+    "@types/node": "^6.0.48",
```

Comment:
Fun open question (this should probably make it into the HQ guide): should `@types` dependencies be in devDependencies or normal dependencies? 

[TypeScript guide](https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html) says `dependencies` for libraries, so downstream TS projects get them, but `devDependencies` if this project is an end-user application/command-line tool/etc, where there is no downstream.